sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicLogicalOperators.scala

```diff
@@ -1145,6 +1145,23 @@ case class Expand(
     copy(child = newChild)
 }
 
+/**
+ * A logical offset, which may removing a specified number of rows from the beginning of the
+ * output of child logical plan.
```

Comment:
Can we mention in this comment:
1. Whether an accompanying LIMIT is also required, or the OFFSET is allowed to stand alone
2. The semantics of which rows get skipped or preserved in the presence of an accompanying LIMIT. For example, if we have LIMIT 10 OFFSET 5, do we (a) impose a total limit of 10 + 5 = 15 rows and then discard the first 5, leaving 10 rows remaining?
3. What are the boundary conditions, e.g. min and max allowed values?

Reply:
Yes. The comments about these semantics already added to `GlobalLimitAndOffset`.